hack/ovn-kind-cno.sh

```diff
@@ -145,22 +149,9 @@ if [ "$BUILD_OVN" = true ] || [ "$BUILD_CNO" = true ]; then
   popd
 fi
 
-count=1
-until kubectl get pod -n openshift-network-operator -o jsonpath='{.items[0].metadata.name}' --field-selector status.phase=Running &> /dev/null ;do
-  if [ $count -gt 15 ]; then
-    echo "Network operator not running"
-    exit 1
-  fi
-  echo "Network operator pod not available yet on attempt $count"
-  count=$((count+1))
-  sleep 10
-
-done
-
-CNO_POD=$(kubectl get pod -n openshift-network-operator -o jsonpath='{.items[0].metadata.name}' --field-selector status.phase=Running)
```

Comment:
`$CNO_POD` is still getting referenced at but undeclared after being deleted https://github.com/openshift/cluster-network-operator/blob/3346964e91c54d8577f5e54425b982701a86118d/hack/ovn-kind-cno.sh#L159 

I added it back in and the script seems to continue ok. Ty.

Reply:
upps, I cleaned too much, it was clearly and oversight on my side 😄

Reply:
recovered